resources/views/collectible/show.blade.php

```diff
@@ -0,0 +1,325 @@
+                        @endphp
+
+                        @if ($meetsRequirement)
+                            <i class="{{ config('other.font-awesome') }} fa-check text-green"></i>
+                        @else
+                            <i class="{{ config('other.font-awesome') }} fa-times text-red"></i>
+                        @endif
+                    </dd>
+                </div>
+            @endforeach
+        </dl>
+    </section>
+
+    <section class="panelV2">
+        <h2 class="panel__heading">Sell requirements</h2>
+        <dl class="key-value">
+            <div class="key-value__group">
+                <dt>Is possessed?</dt>
+                <dd>
+                    @if ($userOwns)
+                        <i class="{{ config('other.font-awesome') }} fa-check text-green"></i>
+                    @else
+                        <i class="{{ config('other.font-awesome') }} fa-times text-red"></i>
+                    @endif
+                </dd>
+            </div>
+            <div class="key-value__group">
+                <dt>Reselling is allowed?</dt>
+                <dd>
+                    @if ($collectible->resell)
+                        <i class="{{ config('other.font-awesome') }} fa-check text-green"></i>
+                    @else
+                        <i class="{{ config('other.font-awesome') }} fa-times text-red"></i>
+                    @endif
+                </dd>
+            </div>
+            <div class="key-value__group">
+                <dt>Is not in stock?</dt>
+                <dd>
+                    @if (! $collectible->inStock())
+                        <i class="{{ config('other.font-awesome') }} fa-check text-green"></i>
+                    @else
+                        <i
+                            class="{{ config('other.font-awesome') }} fa-times text-red"
+                            title="The item must not be in stock in order to resell it"
+                        ></i>
+                    @endif
+                </dd>
+            </div>
+            <div class="key-value__group">
+                <dt>Not already selling?</dt>
+                <dd>
+                    @if (! $userIsSelling)
+                        <i class="{{ config('other.font-awesome') }} fa-check text-green"></i>
+                    @else
+                        <i
+                            class="{{ config('other.font-awesome') }} fa-times text-red"
+                            title="This item already has an active offer to sell."
+                        ></i>
+                    @endif
+                </dd>
+            </div>
+        </dl>
+    </section>
+@endsection
+
+@section('scripts')
+    @vite('resources/js/vendor/chart.js')
+    <script nonce="{{ HDVinnie\SecureHeaders\SecureHeaders::nonce('script') }}">
+        document.addEventListener('DOMContentLoaded', function () {
+            const transactions = {!! Js::encode($transactions) !!};
```

Comment:
https://laravel.com/docs/12.x/blade#rendering-json

Reply:
Changed to `Js::from`.